specification/src/main/asciidoc/platform/Profiles.adoc

```diff
@@ -212,7 +209,7 @@ associated Jakarta Enterprise Beans QL, which have been made optional)
 * Jakarta Security 1.0
 * Jakarta Debugging Support for Other Languages 1.0
 * Jakarta Standard Tag Library 1.2
-* Jakarta Web Services Metadata 2.1
+* Jakarta Web Services Metadata 1.1
```

Comment:
There's some version history here that I'm trying to straighten out.
The JCP version of this spec is version 2.1.  I don't know why this
got set to 1.1 but I'm trying to get that fixed.

Reply:
Since we decided to change the RelatedDocs link to 1.1 (via the note string on the Spec Committee), I'll leave it with 1.1 for now.

Reply:
We fixed all the versions to be 2.1, so this should be updated.